05-Organizing-and-Executing-Simulations/01-Organizing-Data.ipynb

```diff
@@ -0,0 +1,354 @@
+{
```

Comment:
<p>...and remove from here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>